src/memory/app.js

```diff
@@ -0,0 +1,102 @@
+let cards = []
+let temp = ''
+
+let firstCard, secondCard
+
+function clickOnCard(event) {
+    const el = event.target
+    if (!el.classList.contains('matched')) {
+
+        el.classList.add('flip')
+
+        if (temp !== '') {
+            setTimeout(() => {
+                if (el.innerHTML === temp) {
+                    isMatch()
+                } else {
+                    isNotMatch()
+                }
+                temp = ''
+                resetCardValue()
+            }, 1500)
+        } else {
+            temp = el.innerHTML
+        }
+    }
+
+}
+
+function resetCardValue() {
+    [firstCard, secondCard] = [null, null]
+}
+
+function isMatch() {
+    document.querySelector('.flip').classList.add('matched')
+    document.querySelector('.flip').classList.remove('flip')
+    firstCard = this
+
+    document.querySelector('.flip').classList.add('matched')
+    document.querySelector('.flip').classList.remove('flip')
+    secondCard = this
+}
+
+function isNotMatch(){
+    document.querySelector('.flip').classList.remove('flip')
+    document.querySelector('.flip').classList.remove('flip')
+}
+
+function disableCard() {
+    firstCard.removeEventListener('click', clickOnCard)
+    secondCard.removeEventListener('click', clickOnCard)
+    resetCardValue()
+}
+
+document.getElementById('container').addEventListener('click', clickOnCard)
+
+function cardImport() {
+    let cardNum = 8
+    for (let i = 0; i < cardNum; i++) {
+        let card = `http://127.0.0.1:8080//${i}.png`
+        cards.push(card)
+        cards.push(card)
+    }
+    randomCards(cards)
+}
+
+
+function drawGrid() {
+    let gridSize = 16
+    let html = '<div id="list-div">'
+    for (let i = 0; i < gridSize; i++) {
+        html += `<div>`
+        html += `<img id='img'  src = '${cards[i]}' />`
```

Comment:
ovo id='img' znači da ćeš imati duplirane id-eve, plus ničemu ne služi.

Reply:
Trebalo bi da dodaš i alt, kad već sličice nisu na gitu, ja sam morao da dodam alt da bih video kako radi.

Reply:
To su bile klase ali u nekom testiranju sam promenio i zab da vratim. Sredicu sad